changelog.d/pr-7470

```diff
@@ -0,0 +1,8 @@
+synopsis: Suppress warnings for verbosity
+packages: cabal-install
+prs: #7470
+issues: #7286
+
+description: {
+- adds a verbosity flag "+nowarn", to suppress all warnings
+}
```

Comment:
I'd move the description text (description is optional) to the synopsis replacing the current synopsis. I think it's clearer for the user. Same for the other changelog entry